### ISSUE TITLE
N50 - stats file

### ISSUE THREAD

**Issue by analopezlopez** [jtamames/SqueezeMeta#574]
Hi, 
I'm going through some data from old experiments and I'm a little concerned about the N50 of some of my assemblies, because they seem to be oddly high. I'll provide you with an example:

#------------------- Statistics on contigs
#	Assembly
Number of contigs	7552
Total length	239018816
Longest contig	1851814
Shortest contig	215
N50	64576
N90	15394

How does SqueezeMeta pull out this data? I think I read somewhere it does it through Prinseq? I'm not doubting about the calculation itself, just trying to understand what is going on here. Just in case it is relevant, I ran the co-assembly mode on these samples.

Thank you!
Ana

**Comment by jtamames** [comment]
Dear Ana
Yes, the stats are being done with prinseq. What is exactly your concern? You have a low number of contigs that apparently are very long (Very low diversity in your dataset?). Notice that the longest contig is 1.8 Mb, therefore I am not surprised to see a high N50.

**Comment by analopezlopez** [comment]
Yes, it seems to be a low diversity dataset (also confirmed by alpha-diversity indexes). I was just concerned I may have misunderstood the N50 value here but, as you say, it makes sense given the length of the largest contig. I checked and this is consistent with other experiments I'd run previously. Doubt solved. Thanks!
